modes/ebuku/evil-collection-ebuku.el

```diff
@@ -0,0 +1,56 @@
+;;; evil-collection-ebuku.el --- Evil bindings for Ebuku -*- lexical-binding: t -*-
+
+;; Copyright (C) 2020 Alexis
+
+;; Author: Alexis <flexibeast@gmail.com>
+;; Maintainer: Alexis <flexibeast@gmail.com>
+;; URL: https://github.com/emacs-evil/evil-collection
+;; Version: 0.0.1
+;; Package-Requires: ((emacs "25.1"))
+;; Keywords: evil, ebuku, tools
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+;;; Commentary:
+
+;; Evil bindings for Ebuku.
+
+;;; Code:
+
+(require 'ebuku)
+(require 'evil-collection)
+
+
+;;;###autoload
+(defun evil-collection-ebuku-setup ()
+  "Set up `evil' bindings for `Ebuku'."
+  (evil-collection-define-key 'normal 'ebuku-mode-map
+    "q" 'quit-window
+    "a" 'ebuku-add-bookmark
+    "d" 'ebuku-delete-bookmark
+    "e" 'ebuku-edit-bookmark
+    "g" 'ebuku-refresh
+    "n" 'ebuku-next-bookmark
```

Comment:
Maybe "j" or "C-j"?

I can see an evil user wanting to use "n" for when they're searching for text.

Reply:
Changed "n" to "j", and added "C-j" as well.